public/js/monitor.js

```diff
@@ -196,6 +191,19 @@ async function hashEmailAndSend(emailFormSubmitEvent) {
   emailForm.submit();
 }
 
+const resendSubscribeData = function() {
+  document.getElementById("confirm-your-account").classList.add("sending");
+  const userEmail = {
+    "email": document.getElementById("submitted-email").innerText,
+  };
+  postData("/user/add", userEmail)
+    .then(data => {
+      document.getElementById("confirm-your-account").classList.add("sent");
+      document.getElementById("resend-data").removeEventListener("click", resendSubscribeData);
+    })
+  .catch(error => console.error(error));
```

Comment:
If there is an error, are we showing any messages or anything to the user, or failing silently here?

Reply:
Failing silently right now, need UX for what happens on fails and resubmit events.